bundles/org.openhab.automation.jrubyscripting/README.md

```diff
@@ -88,56 +130,42 @@ Additional [example rules are available](https://openhab.github.io/openhab-jruby
   - Designed and tested using [Test-Driven Development](https://en.wikipedia.org/wiki/Test-driven_development) with [RSpec](https://rspec.info/)
 - Extensible.
   - Anyone should be able to customize and add/remove core language features
-- Easy access to the Ruby ecosystem in rules through Ruby gems.
+- Easy access to the Ruby ecosystem in rules through [Ruby Gems](https://rubygems.org/).
 
 ## Installation
 
-### Prerequisites
-
-1. openHAB 3.4+
-1. The JRuby Scripting Language Addon
-
 ### From the User Interface
 
-1. Go to `Settings -> Add-ons -> Automation` and install the jrubyscripting automation addon following the [openHAB instructions](https://www.openhab.org/docs/configuration/addons.html).
-   In openHAB 4.0+ the defaults are set so the next step can be skipped.
-1. Go to `Settings -> Add-on Settings -> JRuby Scripting`:
-   - **Ruby Gems**: `openhab-scripting=~>5.0`
-   - **Require Scripts**: `openhab/dsl` (not required, but recommended)
+- Go to `Settings -> Add-ons -> Automation` and install the jrubyscripting automation addon following the [openHAB instructions](https://www.openhab.org/docs/configuration/addons.html).
 
 ### Using Files
 
-1. Edit `<OPENHAB_CONF>/services/addons.cfg` and ensure that `jrubyscripting` is included in an uncommented `automation=` list of automations to install.
-   In openHAB 4.0+ the defaults are set so the next step can be skipped.
-1. Configure JRuby openHAB services
-
-   Create a file called `jruby.cfg` in `<OPENHAB_CONF>/services/` with the following content:
-
-   ```ini
-   org.openhab.automation.jrubyscripting:gems=openhab-scripting=~>5.0
-   org.openhab.automation.jrubyscripting:require=openhab/dsl
-   ```
+- Edit `<OPENHAB_CONF>/services/addons.cfg` and ensure that `jrubyscripting` is included in an uncommented `automation=` list of automations to install.
 
 ## Configuration
 
 After installing this add-on, you will find configuration options in the openHAB portal under _Settings -> Add-on Settings -> JRuby Scripting_.
 Alternatively, JRuby configuration parameters may be set by creating a `jruby.cfg` file in `conf/services/`.
 
+> **_NOTE:_**
+> In openHAB 3.4.x, the `gems` and `require` settings must be manually configured to the value given in the table below.
+Starting from openHAB 4.0, the correct defaults were added, so manual configurations are no longer necessary.
```

Comment:
I'm not sure we should manage versioning in this README which will be for 5.0. The 3.4 documentation can be found here: https://v34.openhab.org/addons/automation/jrubyscripting/

If this is important as something to be aware of when upgrading, we can keep it, but if its target audience is only 3.4 users, I think we should omit it. WDYT?

Reply:
At the moment, the library concurrently supports all versions from 3.4 all the way to 5.0. This document is generated from the master document, hence why it's here. It can be removed manually of course, but prefer not to have to do it. When the support for 3.4 is dropped, this part will be removed.

@ccutrer would you like to chime in?

Reply:
I've manually removed the note

Reply:
OK, hope it isn't too much hassle. Would have merged anyway, but was waiting for @ccutrer's response.